### ISSUE TITLE
I need help I have no idea what I just did

### ISSUE THREAD

**Issue by Phoenixhahaha** [Monika-After-Story/MonikaModDev#8346]
each time I go on MAS its just says a exception has occurred 

heres the traceback  

While loading <'Image' u'mod_assets/location/Spaceroom V3.1/V3.1-n.png'>:
  File "game/script-ch30.rpy", line 1510, in script call
  File "game/event-handler.rpy", line 2530, in script call
  File "game/script-story-events.rpy", line 1185, in script call
  File "game/script-story-events.rpy", line 1440, in script call
  File "game/script-ch30.rpy", line 998, in script
  File "game/script-ch30.rpy", line 998, in <module>
  File "game/script-ch30.rpy", line 438, in mas_drawSpaceroomMasks
  File "game/definitions.rpy", line 216, in mas_with_statement
IOError: Couldn't find file 'mod_assets/location/Spaceroom V3.1/V3.1-n.png'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script-ch30.rpy", line 1510, in script call
  File "game/event-handler.rpy", line 2530, in script call
  File "game/script-story-events.rpy", line 1185, in script call
  File "game/script-story-events.rpy", line 1440, in script call
  File "game/script-ch30.rpy", line 998, in script
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script-ch30.rpy", line 998, in <module>
  File "game/script-ch30.rpy", line 438, in mas_drawSpaceroomMasks
  File "game/definitions.rpy", line 216, in mas_with_statement
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/core.py", line 2096, in do_with
    clear=clear)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/core.py", line 2526, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/core.py", line 2883, in interact_core
    self.draw_screen(root_widget, fullscreen_video, (not fullscreen_video) or video_frame_drawn)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/core.py", line 1955, in draw_screen
    renpy.config.screen_height,
  File "renpy/display/render.pyx", line 427, in renpy.display.render.render_screen (gen/renpy.display.render.c:6806)
    rv = render(root, width, height, 0, 0)
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen/renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/layout.py", line 693, in render
    surf = render(child, width, height, cst, cat)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen/renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen/renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/transition.py", line 361, in render
    top = render(self.new_widget, width, height, st, at)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen/renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen/renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/layout.py", line 693, in render
    surf = render(child, width, height, cst, cat)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen/renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen/renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/layout.py", line 693, in render
    surf = render(child, width, height, cst, cat)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen/renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen/renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "renpy/display/accelerator.pyx", line 108, in renpy.display.accelerator.transform_render (gen/renpy.display.accelerator.c:2027)
    cr = render(child, widtho, heighto, st - self.child_st_base, at)
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen/renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/image.py", line 387, in render
    return wrap_render(self.target, width, height, st, at)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/image.py", line 208, in wrap_render
    rend = render(child, w, h, st, at)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen/renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen/renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/im.py", line 473, in render
    im = cache.get(self)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/im.py", line 200, in get
    surf = image.load()
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/display/im.py", line 524, in load
    surf = renpy.display.pgrender.load_image(renpy.loader.load(self.filename), self.filename)
  File "/Users/choloeflores/Library/Application Support/Steam/steamapps/common/Doki Doki Literature Club/renpy/loader.py", line 543, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'mod_assets/location/Spaceroom V3.1/V3.1-n.png'.

Darwin-19.6.0-x86_64-i386-64bit
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.4
[/code]

**Comment by dreamscached** [comment]
Some of the game assets are missing. I'd advise you *against* using DDLC from Steam since it's know to cause issues (in your case it seemed to remove mod files.)

You most likely won't fix this on your own, so reinstall the mod using non-Steam version from [here](https://ddlc.moe/).

**Comment by ThePotatoGuy** [comment]
seems a submod is missing assets? You need to contact the submod creator for assistance.

**Comment by ThePotatoGuy** [comment]
[closing as resolved]
